Automattic/jetpack
# Stats: Add image to upgrade notice

Part of #28729.

## Proposed changes:


Adds an image + gradient to the upgrade nudge.

<img width="860" alt="SCR-20230306-keh" src="https://user-images.githubusercontent.com/40267301/223037270-9f86d5aa-06af-4694-8a30-4c138d10e3d7.png">

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Make sure Odyssey Stats are disabled.
- Visit Jetpack → Stats.
- Confirm the image is present in the upgrade nudge.
- Confirm the image is removed at smaller screen sizes.

**Looking for feedback!**

I'm not super happy with the gap below the image when resizing the screen but I'm unsure of how to remove it and have the image size itself properly. I've tried setting both images via CSS but then neither are drawn. I need something in the HTML to trigger proper layout.

**Updated:(per feedback)**

- Images are now handled strictly via CSS.
- Font sizes and padding adjusted.
- Missing translation calls have been added back.
- Updated image in PR description to match current status of the notice.
